public/app/plugins/datasource/loki/addToQuery.ts

```diff
@@ -39,22 +39,36 @@ export function addLabelToQuery(query: string, key: string, operator: string, va
   }
 }
 
-type StreamSelectorPosition = { from: number; to: number; query: LokiVisualQuery };
```

Comment:
These have been unified into `type Position = { from: number; to: number };` as we don't need query.